card/mod/05_standard/set/self/head.rb

```diff
@@ -13,79 +12,93 @@
 
   view :core do |args|
     case
-    when focal?    ; CGI.escapeHTML _render_raw(args)
-    when @mainline ; "(*head)"
-    else           ; _render_raw(args)
+    when focal?    then CGI.escapeHTML _render_raw(args)
+    when @mainline then "(*head)"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.